client/components/page-wrapper.jsx

```diff
@@ -5,7 +5,9 @@ export default function PageWrapper(props) {
     ? 'bg-pattens-blue'
     : props.path === ''
       ? 'bg-white'
-      : 'bg-venice-blue';
+      : props.path === 'search'
+        ? 'bg-white'
+        : 'bg-venice-blue';
```

Comment:
This ternary is starting to get out of hand. Please, make a method or helper function that takes a path as a parameter and returns a CSS class using normal `if` statements.